README.md

```diff
@@ -358,23 +358,28 @@ extern "Rust" {
     /// `ptr` has to point to the beginning of an allocated block.
     fn miri_static_root(ptr: *const u8);
 
+    // Miri-provided extern function to get the amount of frames in the current backtrace.
+    // The `flags` argument must be `1`.
+    fn miri_backtrace_size(flags: u64) -> usize;
+
     /// Miri-provided extern function to obtain a backtrace of the current call stack.
-    /// This returns a boxed slice of pointers - each pointer is an opaque value
+    /// This writes a slice of pointers into `buf` - each pointer is an opaque value
     /// that is only useful when passed to `miri_resolve_frame`
-    /// The `flags` argument must be `0`.
-    fn miri_get_backtrace(flags: u64) -> Box<[*mut ()]>;
+    /// `miri_backtrace_size` must be used first to get the amount of space to allocate for `buf` and
```

Comment:
This should say how much space there needs to be (namely, `miri_backtrace_size(1) * pointer_size`)